Vagrantfile

```diff
@@ -23,6 +23,19 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.provision "file", source: File.join(Dir.home, ".gitconfig"), destination: ".gitconfig"
   end
 
+  # Use the boto configuration from the host in the VM, if it is in the
+  # expected location
+  if File.exists?(File.join(Dir.home, ".boto"))
+    config.vm.provision "file", source: File.join(Dir.home, ".boto"), destination: ".boto"
+  end
+
+  if ENV.has_key?('VERSION')
```

Comment:
> We can either drop mention of Vagrantfile from docs for 3rd party developers, or have two Vagrantfiles.

I'll have a look into using a different Vagrantfile.

> Separately, copying over environment variables is super-brittle-magic and not a good idea (what if the user decides they prefer ZSH and immediately switch to that?).

The release process is not guaranteed to work if it is not followed exactly (or if it is :disappointed: ).